src/doom/d_main.c

```diff
@@ -1384,6 +1385,11 @@ void D_DoomMain (void)
 	sidemove[1] = sidemove[1]*scale/100;
     }
     
+    if (M_CheckParm("-pistolstart"))
```

Comment:
Nit: you can simplify this to just:

    pistolstart = M_ParmExists("-pistolstart");